### PR TITLE
Add integration coverage for multiline agent paste

### PR DESCRIPTION
**Pull request by haonanttt**
## Summary

- Add an end-to-end test for the multiline agent-pane paste flow introduced in #415.
- Exercise the real OS clipboard and window-level Ctrl+V path.
- Verify all pasted lines remain in one bordered agent draft and are not submitted into the transcript.
- Tighten the existing single-line paste assertion to require the marker inside the input row.
- Map release checklist item C065 specifically to the multiline test.

## Coverage

The integration test covers:

Ctrl+V -> TerminalPage agent-pane interception -> `agent_paste_text` protocol event -> WTA clipboard read -> multiline draft insertion

Deterministic edge cases such as normalization, routing, modal gating, stale generations, and failure cleanup remain covered by existing Rust unit tests.

## Stability and side effects

- Never presses Enter or submits a Copilot prompt.
- Uses random markers to avoid stale-text matches.
- Cleans the draft in `finally`.
- Treats only explicit foreground-acquisition failures as skipped.
- Uses one pane capture to atomically validate the complete multiline draft.
- Keeps the existing clipboard-overwrite behavior used by the paste E2E suite.

## Testing

```powershell
Invoke-Pester test/e2e/tests/Feature.Paste.Tests.ps1 -Tag Feature
```

Result: 2 passed, 0 failed, 0 skipped.

### CODE REVIEW

File: test/e2e/release-coverage-map.psd1 [hunk 1]
```diff
@@ -106,8 +106,8 @@
     'Bug report zip includes agent logs' = 'Bug report zip includes agent logs'
     # §10 diagnostics — hook trace log (Feature.HookTrace). A hooked copilot prompt is traced.
     'Hook trace log is written'         = 'Hook trace log is written'
-    # §2 agent pane paste (Feature.Paste) — clipboard text pasted with Ctrl+V lands in the input.
-    'Paste works'                       = 'Paste works'
+    # §2 agent pane paste (Feature.Paste) — only the multiline case satisfies C065.
+    'Paste works'                       = '^Paste works \(multiline clipboard text stays in one agent draft without submitting\)$'
```
[the remark this answers is not in the document]
Comment (haonanttt) [reply test/e2e/release-coverage-map.psd1]: Fixed in d53801563. The NUnit XML uses the full name (`Feature §2 agent pane paste.<It name>`), so I removed the anchors while retaining the unique multiline test substring. I also verified the updated regex matches exactly one real test-case name and that New-ReleaseReport marks C065 as passed.

**Review by DDKinger — APPROVED** [review]
